### PR TITLE
Release/beta 1 5

### PR DESCRIPTION
**Pull request by swienribGenesys**

### CODE REVIEW

File: GCMessengerSDKSample/GCMessengerSDKSample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -421,7 +439,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 9;
+				CURRENT_PROJECT_VERSION = 1;
```
Comment (tehilaR2) [inline GCMessengerSDKSample/GCMessengerSDKSample.xcodeproj/project.pbxproj:442]: the version number restarts for each version?

File: GCMessengerSDKSample/Podfile.lock
Comment (tehilaR2) [inline GCMessengerSDKSample/Podfile.lock:1]: Should this file be on the PR?

**Comment by tehilaR2** [comment]
@swienribGenesys  Please resolve conflicts

File: GCMessengerSDKSample/Podfile.lock
Comment (swienribGenesys) [reply GCMessengerSDKSample/Podfile.lock:1]: Yes

**Review by tehilaR2 — APPROVED** [review]

**Review by SzaszaGenesys — COMMENTED** [review]
Test

**Pull request merged**